Dockerfile

```diff
@@ -32,7 +32,7 @@ COPY pkg/ pkg/
 COPY cmd/ cmd/
 COPY third_party/ third_party/
 
-RUN mkdir bin; CGO_ENABLED=0 go build -o bin ./cmd/...
+RUN mkdir bin; CGO_ENABLED=0 go build -ldflags "-X k8s.io/component-base/version.gitVersion=v1.23.4" -o bin ./cmd/...
```

Comment:
This feels brittle as-is... someone is going to have to remember to update this when we rebase Kubernetes, it's only applicable when doing a Dockerfile-based build, etc.

I'd suggest we discuss how to proceed in an issue. Thanks!

Reply:
Yes I do that. Thanks